app/src/main/java/androidx/fragment/app/FragmentStatePagerAdapterMenuWorkaround.java

```diff
@@ -127,8 +132,7 @@ public FragmentStatePagerAdapterMenuWorkaround(@NonNull final FragmentManager fm
     }
 
     /**
-     * @param position the position of the item you want
-     * @return the {@link Fragment} associated with a specified position
+     * Return the Fragment associated with a specified position.
```

Comment:
Why this change? It is causing the build to fail

Reply:
Simply figured that I'd move over to the newer changes as much as possible, though I guess the documentation doesn't really matter that much. If it's causing checkstyle errors, I'll just change it back.